Makefile

```diff
@@ -171,3 +175,8 @@ verify-bundle: bundle
 .PHONY: bundle-build
 bundle-build:
 	$(CONTAINER_ENGINE) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+# Push the bundle image to the registry
+.PHONY: bundle-push
+bundle-push:
+	$(CONTAINER_ENGINE) push $(BUNDLE_IMG)
```

Comment:
A few thoughts:

1. We should have `bundle-build` depend on `bundle` to ensure that we we have the most up to date manifests
2. I think it is OK for `bundle-push` to depend on `bundle-build` - that way `make bundle-push` can act as our release command.